src/utils/errorResponse.js

```diff
@@ -0,0 +1,15 @@
+import { HTTP_STATUS, MESSAGES } from "../constants/index.js";
+
+export const respondInvalidFormat = (res) => {
+  return res.status(HTTP_STATUS.BAD_REQUEST).json({
+    status: HTTP_STATUS.BAD_REQUEST,
+    error: MESSAGES.ERROR.INVALID_FORMAT,
+  });
+};
+
+export const respondNotFound = (res, message) => {
+  return res.status(HTTP_STATUS.NOT_FOUND).json({
+    status: HTTP_STATUS.NOT_FOUND,
+    error: message,
+  });
+};
```

Comment:
```suggestion
const respondWithError = (res, status, message) => {
  return res.status(status).json({ status, error: message });
};

export const respondInvalidFormat = (res) => {
  respondWithError(res, HTTP_STATUS.BAD_REQUEST, MESSAGES.ERROR.INVALID_FORMAT);
};

export const respondNotFound = (res, message) => {
  respondWithError(res, HTTP_STATUS.NOT_FOUND, message);
};
```
`respondInvalidFormat`, `respondNotFound` 함수를 분리하여 응답 중복 코드 제거를 잘 해주셨네요!

다만, 현재 두 함수 모두 `res.status().json({ status, error })` 패턴이 반복되고 있어서
제안드린 것처럼 공통 응답 헬퍼 함수(`respondWithError`)로 분리하면 중복을 줄이고 응답 구조를 한 곳에서 관리할 수 있어 유지보수에 유리할 것 같아요. :)

Reply:
피드백 감사합니다!
말씀해주신 대로 응답 패턴이 반복되고 있어서 `respondWithError` 함수로 공통 분리해서 반영하였습니다.
좋은 리뷰 덕분에 더 깔끔하게 유지보수할 수 있을 것 같아요! 😊